### PR TITLE
 added new  "code" button

### PR DESCRIPTION
**Pull request by tmott13**
Added a new button style under Special buttons called Code Button

Issue: Add New Buttons #1

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -516,6 +516,16 @@ $(document).ready(function () {
             },
           ],
         },
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:519]: Place this in the correct alphabetical order under Special

**Review by shahednasser — CHANGES_REQUESTED** [review]
It looks nice but can you add some action to it on click? just a little style change on `:active` maybe simple box shadow or outline.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Also please remove the "sbuttons" submodule you added

File: assets/js/buttons-examples.js (hunk 1 above)
Comment (tmott13) [reply assets/js/buttons-examples.js]: done in my recent commit

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @tmott13 for code and idea

**Pull request merged**